decidim-core/app/mailers/decidim/application_mailer.rb

```diff
@@ -17,6 +17,7 @@ def set_smtp
       return if @organization.nil? || @organization.smtp_settings.blank?
 
       mail.from = @organization.smtp_settings["from"].presence || mail.from
+      mail.reply_to = Decidim.config.mailer_reply || mail.from
```

Comment:
Solved here : 18e405b

Reply:
@Dynnammo Is there anre reason you want to use `mail.from` for the `reply_to` address?

We bumped into this line when we realized that the `reply_to` address we set in a custom mailer is not being applied because this line will always override it.

I think we should respect the `reply_to` address if the mailer wants to set it.

Reply:
Sorry for the delay. I'll take the time to investigate on it by the end of the week.

Reply:
@Dynnammo No worries!

We are already at it. Please take a look at: https://github.com/decidim/decidim/pull/7641